test/integration/targets/azure_rm_virtualmachine_scaleset/tasks/main.yml

```diff
@@ -151,62 +150,140 @@
   assert:
     that: results.changed
 
-- name: Create VMSS (check mode)
+- name: Delete VMSS
   azure_rm_virtualmachine_scaleset:
     resource_group: "{{ resource_group }}"
     name: testVMSS{{ rpfx }}1
+    state: absent
+    remove_on_absent: ['all']
     vm_size: Standard_DS1_v2
+    capacity: 2
+    image:
+      offer: CoreOS
+      publisher: CoreOS
+      sku: Stable
+      version: latest
+
+- name: Create VMSS with security group in same resource group, with accelerated networking.
+  azure_rm_virtualmachine_scaleset:
+    resource_group: "{{ resource_group }}"
+    name: testVMSWithAccNetworking
```

Comment:
Please use a random number for the name like `testVMSS{{ rpfx }}2` to avoid name conflict when parallel running test